addons/iap/models/iap_account.py

```diff
@@ -18,17 +18,101 @@ class IapAccount(models.Model):
     _rec_name = 'service_name'
     _description = 'IAP Account'
 
-    service_name = fields.Char()
-    account_token = fields.Char(default=lambda s: uuid.uuid4().hex)
+    service_name = fields.Char(string='Account Name')
+    account_token = fields.Char(default=lambda s: uuid.uuid4().hex,
+                                help="Account token is your authentication key for this service. Do not share it.",
+                                size=43)
```

Comment:
```suggestion
    account_token = fields.Char(
        default=lambda s: uuid.uuid4().hex,
        help="Account token is your authentication key for this service. Do not share it.",
        size=43)
```
This prevents the trouble of maintaining the alignment between the parenthesis of the first lines and the other lines. It's also easier to read IMO.